src/components/googleLogInBtn.jsx

```diff
@@ -10,17 +11,31 @@ class GoogleLogInBtn extends Component {
     var googleId = googleUser.getId();
 
     let profile = googleUser.getBasicProfile();
-    firebase
+    var stringPart = profile.getEmail().split('@');
+    var userInfo = firebase.database().ref('users/' + localStorage.getItem('id'));
```

Comment:
this is why you shouldnt clear local storage on logout. sincee we need local storage here